en/docs/design/create-api/create-api-revisions.md

```diff
@@ -0,0 +1,89 @@
+
+     [![Delete revision if max count is reached]({{base_path}}/assets/img/design/revision/delete-and-create-revision.png)]({{base_path}}/assets/img/design/revision/delete-and-create-revision.png)
+
+## Deploy a revision
+
+Follow the instructions below to either deploy a new revision or deploy an existing revision in a given Gateway Environment:
+
+### Deploy a new revision
+
+Follow the steps below if you need to **deploy a new revision** on a given Gateway Environment:
+
+1. Sign in to the Publisher.
+
+     [https://localhost:9443/publisher](https://localhost:9443/publisher)
+
+2. Click **Deployments**.
+
+3. Click **Deploy New Revision**.
+
+     [![Deploy a new revision]({{base_path}}/assets/img/design/revision/deploy-new-revision.png)]({{base_path}}/assets/img/design/revision/deploy-new-revision.png)
+
+4. Provide a description for the new revision.
+5. Select one or more Gateway Environments from the environments listed.
+6. Click **Deploy**.
+
+     [![Deploy a new revision view]({{base_path}}/assets/img/design/revision/deploy-new-revision-example.png)]({{base_path}}/assets/img/design/revision/deploy-new-revision-example.png)
+
+### Deploy an existing revision
+
+Follow the instructions below if you need to **deploy an existing revision** on a given Gateway Environment:
+
+1. Sign in to the Publisher.
+
+     [https://localhost:9443/publisher](https://localhost:9443/publisher)
+
+2. Click **Deployments**.
+3. Select the required revision from the **Deployed Revision** drop-down menu.
+4. Click on the **Deploy** button.
+
+     [![Deploy an existing revision]({{base_path}}/assets/img/design/revision/deploy-existing-revision.png)]({{base_path}}/assets/img/design/revision/deploy-existing-revision.png)
+
+## Undeploy a revision
+
+Follow the instructions below if you need to undeploy a revision from a Gateway Environment:
+
+1. Sign in to the Publisher.
+
+     [https://localhost:9443/publisher](https://localhost:9443/publisher)
+
+2. Click **Deployments**.
+3. Click on the **Undeploy** button that appears against the respective Gateway Environment from which you want to undeploy the revision.
+
+     [![Undeploy a revision]({{base_path}}/assets/img/design/revision/undeploy-revision.png)]({{base_path}}/assets/img/design/revision/undeploy-revision.png)
+
+## Delete a revision
+
+If you need to delete a specific revision, click on the **Delete** button in the revision list as shown below. If a revision is already deployed in a specific environment, you cannot delete it. You should first undeploy the revision from the Gateway Environment and then delete it.
+
+[![Delete a revision]({{base_path}}/assets/img/design/revision/delete-revision.png)]({{base_path}}/assets/img/design/revision/delete-revision.png)
+
+## Restore a revision
+
+If you need to restore a specific revision, click on the **Restore** button in the revision list as shown below. This will restore the revision in to the working copy.
```

Comment:
Is the terminology "working copy" correct here? Or has a new name been decided on?

Reply:
Yes this should be changed.

Reply:
Shall we change it to ```This will restore the API with the specific revision.```?

Reply:
@tharikaGitHub  +1 to the suggested change